app/Helpers/DatabaseHelper.php

```diff
@@ -7,41 +7,41 @@
 use App\Models\GamesFile;
 use Carbon\Carbon;
 
-class DatabaseHelper{
+class DatabaseHelper {
 
-    public static function getReleaseDateFromGameId($gameid){
+    public static function getReleaseDateFromGameId($gameid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->